packages/manager/modules/pci/src/projects/project/kubernetes/details/audit-logs/audit-logs.html

```diff
@@ -1,18 +1 @@
-<div>
-    <span data-translate="kube_logs_description"></span>
-    <a
-        data-ng-href="{{::$ctrl.getLogsInfoLink()}}"
-        target="_blank"
-        rel="noopener"
-    >
-        <span data-translate="kube_logs_description_link"></span>
-        <span class="oui-icon oui-icon-external-link" aria-hidden="true"></span>
-    </a>
-
-    <log-to-customer
-        data-log-keys="$ctrl.LOG_KEYS"
-        data-log-kind-api-url="$ctrl.logKindApiUrl"
-        data-log-api-url="$ctrl.logApiUrl"
-    >
-    </log-to-customer>
-</div>
+<div></div>
```

Comment:
still used ?

Reply:
No, but I thought I could do a PR dedicated to the clean of audit-logs part to keep this PR as small as possible